src/views/HelpAndKnowledge/Articles/_components/RelatedArticles/index.tsx

```diff
@@ -5,47 +5,64 @@ import { CardGroup, GridContainer } from '@trussworks/react-uswds';
 import classnames from 'classnames';
 
 import helpAndKnowledgeArticles, {
+  ArticleProps,
   ArticleTypeProps
 } from 'views/HelpAndKnowledge/Articles';
 import ArticleCard from 'views/HelpAndKnowledge/Articles/_components/ArticleCard';
 
 type RelatedArticlesProps = {
   className?: string;
   currentArticle: string;
-  viewAllLink?: boolean;
+  specificArticles?: string[];
```

Comment:
We can defined an array of fixed length in typescript using tuple type.  We then don't have to do a check on the length below

```
specificArticles?: [string, string, string];
```